### PR TITLE
New PRS in EMML1832 (no.3)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a number of new records for personal names in EMML1832 and updated some of the existing ones.

### CODE REVIEW

File: new/PRS14246AmlakMoa.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">አምላክ፡ ሞአ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾAmlāk Moʾa</persName>
+                    <floruit notBefore="1270" notAfter="1340">13th to 14th century</floruit>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS14246AmlakMoa" passive="EMML1832"/>
+                    <relation name="dcterms:relation"><desc>Adressed as 'my father' by <persName ref="PRS10374Yohanne"/>.</desc></relation>
```
Comment (eu-genia) [inline new/PRS14246AmlakMoa.xml:54]: a relation must always have an active and a passive element

File: new/PRS14248KrestosMekhena.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">Krǝstos Mǝkḥǝna</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <floruit notBefore="1250" notAfter="1360">13th to 14th century</floruit>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS14248KrestosMekhena" passive="EMML1820"/>
+                    <relation name="dcterms:relation"><desc>Adressed as 'my father' by <persName ref="PRS13230Bakimos"/>.</desc></relation>
```
Comment (eu-genia) [inline new/PRS14248KrestosMekhena.xml:54]: active passive

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: PRS13001-14000/PRS13198Tewodros.xml [hunk 3]
```diff
@@ -47,18 +47,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">ቴዎድሮስ</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Tewodros</persName>
+                    <persName xml:lang="gez"><roleName type="title">kāhǝn</roleName></persName>
                     <faith type="EOTC"/>
                     <occupation type="literary"/>
+                    <occupation type="ecclesiastic">manakos</occupation>
```
Comment (thea-m) [inline PRS13001-14000/PRS13198Tewodros.xml:53]: I think that giving the English "monk" might be appropriate here. I don't feel that the Geez term has technical meanings not covered by the English one.

File: PRS9001-10000/PRS9199TaklaYo.xml [hunk 4]
```diff
@@ -45,12 +46,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <body>
          <listPerson>
             <person sex="1">
-               <persName>
-                  
-                  Täklä Yoḥannǝs
-               </persName>
-               <occupation> mähari </occupation>
+               <persName xml:lang="gez" xml:id="n1">ተክለ፡ ዮሐንስ፡</persName>
+               <persName xml:lang="gez" corresp="#n1" type="normalized">Täklä Yoḥannǝs</persName>
```
Comment (thea-m) [inline PRS9001-10000/PRS9199TaklaYo.xml:50]: Takla

File: new/PRS14248KrestosMekhena.xml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-01-10">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">Krǝstos Mǝkḥǝna</persName>
```
Comment (thea-m) [inline new/PRS14248KrestosMekhena.xml:48]: fidal missing, transcription and fidal to be swapped

File: new/PRS14249Benyam.xml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-01-10">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">Bǝnyām</persName>
```
Comment (thea-m) [inline new/PRS14249Benyam.xml:48]: same as above

**Review by thea-m — CHANGES_REQUESTED** [review]
Cool, thank you!

File: PRS9001-10000/PRS9199TaklaYo.xml [hunk 7]
```diff
@@ -45,12 +46,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+               <occupation type="ecclesiastic">mahari </occupation>
```
Comment (eu-genia) [inline PRS9001-10000/PRS9199TaklaYo.xml:51]: not mahāri?

File: PRS13001-14000/PRS13198Tewodros.xml [hunk 8]
```diff
@@ -47,18 +47,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">ቴዎድሮስ</persName>
```
Comment (eu-genia) [inline PRS13001-14000/PRS13198Tewodros.xml:48]: : at the end

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: PRS13001-14000/PRS13198Tewodros.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply PRS13001-14000/PRS13198Tewodros.xml:53]: Yes.

File: PRS9001-10000/PRS9199TaklaYo.xml (hunk 7 above)
Comment (CarstenHoffmannMarburg) [reply PRS9001-10000/PRS9199TaklaYo.xml:51]: The manuscript on folio 1va gives marāhi. Is it a mistake by the scribe?

Screenshot 2024_marahi Takla Yohannes

Comment (eu-genia) [reply PRS9001-10000/PRS9199TaklaYo.xml:51]: it is a different word, both exist. then write marāhi

Comment (CarstenHoffmannMarburg) [reply PRS9001-10000/PRS9199TaklaYo.xml:51]: Yes, marāhi / marāḥi = 'leader' should be a possible term in that place.

**Comment by CarstenHoffmannMarburg** [comment]
If all requested changes are met, can this branch be approved and merged?

**Review by eu-genia — APPROVED** [review]

**Pull request merged**